### PR TITLE
User ID mismatch issue

### PR DESCRIPTION
**Pull request by Rizayev**
It appears that the user IDs are not aligning correctly, leading to inconsistencies in the data. This pull request aims to address this issue by ensuring that user IDs are synchronized across the system.

### CODE REVIEW

**Comment by rupadana** [comment]
@Rizayev i think its better to hide users selector if the checkbox is checked, can you update it?

File: src/Resources/AnnouncementResource.php [hunk 1]
```diff
@@ -50,13 +51,14 @@ public static function form(Form $form): Form
...
 
-                Select::make('users')
-                    ->options([
-                        'all' => 'all',
-                        ...User::all()->pluck('name', 'id'),
-                    ])
+                 Select::make('users')
+                    ->options(User::pluck('name', 'id')->toArray())
                     ->multiple()
                     ->required(),
```
Comment (rupadana) [inline src/Resources/AnnouncementResource.php]: Hide this component if Checkbox 'All users' is checked

Comment (Rizayev) [reply src/Resources/AnnouncementResource.php]: Yes sure

**Comment by rupadana** [comment]
@Rizayev Thank you for your contribution. I appreciate it.

**Pull request merged**